src/pyOpenMS/pyTOPP/MRMMapper.py

```
@@ -8,56 +8,75 @@
+                    elif targeted.hasNucleotide(transition.getTransGroupRef()):
+                        this_sequence = targeted.getCompoundByRef(transition.getTransGroupRef()).sequence
+                        sequence_type = "nucleotide"
+                    else:
+                        this_sequence = "unknown"
+                        sequence_type = "unknown"
+                    
+                    other_sequence = chrom.getPrecursor().getMetaValue("transition_group_name")
+                    print("Found mapping of", chrom.getPrecursor().getMZ(), "/", chrom.getProduct().getMZ(), "to", transition.getPrecursorMZ(), "/",transition.getProductMZ())
+                    print("Of", sequence_type, this_sequence)
+                    print("But the chromatogram is already mapped to", other_sequence)
                     if not allow_double_mappings: raise Exception("Cannot map twice")
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)